software/holoplay.yml

```diff
@@ -0,0 +1,12 @@
+name: "HoloPlay"
+website_url: "https://app.holoplay.io"
+source_code_url: "https://github.com/stephane-r/holoplay-pwa"
+description: "Web app using Invidious API for listening Youtube audio source."
+licenses:
+  - MIT
+platforms:
+  - Docker
+  - React
```

Comment:
```suggestion
  - Nodejs
```

Reply:
Mmm, i'm not sur for this change. It's not an Node.js application. It's an client only app using React. The server on Dockerfile is an basic nginx image.

Of cours, it use Node.js, but only for compilation process, not on production environment.

Reply:
Personally I would be okay with Node because react uses node.

Reply:
> It's an client only app using React. The server on Dockerfile is an basic nginx image.

I understand what you mean, but at least as far as I can see from your Getting Started section, there are two main deployment methods for the Software - utilizing npm in development mode (which implies Nodejs), or employing a docker image. Hence, for me, the principal server-side prerequisites are Nodejs or Docker, given that React is not necessary for the server-side deployment

Reply:
I've no problem to add Node.js :+1: 

For the production server : you can use Ruby, PHP, or other server you like. The production build is an HTML, CSS and JS files :) Node.js is only required for the compilation process.

Reply:
Just to add to this https://github.com/stephane-r/holoplay-pwa?tab=readme-ov-file#setup lists npm making it a requirement.

Reply:
Node.js and npm is require only for build step or development environment. You can use HoloPlay without build by using release source or Docker image.

I'll update the README for add this detail 👍